src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -389,7 +389,12 @@ def _get_pipeline_class(
         return class_obj
 
     diffusers_module = importlib.import_module(class_obj.__module__.split(".")[0])
-    class_name = config["_class_name"]
+    class_name = class_name or config["_class_name"]
```

Comment:
Is that fully backwards compatible?

Reply:
I searched the code base and it seems like `class_name` or config is always passed here.